articles/building-a-simple-angular-app/index.md

```diff
@@ -0,0 +1,361 @@
+Angular 11 was released on Nov 11, 2020, by the Angular team at Google.
+Angular's development community has been growing over the past few years.
+If you aren't familiar with Angular, there are plenty of reasons why you should start learning.
+This year's release focuses on improving the developer's experience.
+To check out more about releases, [click here](#https://blog.angular.io/version-11-of-angular-now-available-74721b7952f7).
+
+### Introduction 
+This article will walk you by building a simple Angular application covering the basic and core Angular CLI concepts.
+The guide is handy for developers who are willing to have a hands-on experience, along with learning.
+
+We are going to build a tiny version of a streaming anime application. On the home page,
+we will display all the anime covers. The user can click on any of the cards to be redirected to the respective anime profile section.
+
+The anime profile contains all the details about that respective anime—
+also, a comment box at the bottom.
+### Getting Started
+
+With Node.js installed, run the following command to install Angular CLI.
```

Comment:
Hi, @MahanteshR, redirect users with no prior Node.js knowledge to a page with installation instructions.